packages/ic-signature-verification/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "ic-signature-verification"
```

Comment:
Apologies if this was discussed earlier, but the difference with the existing [ic-crypto-standalone-sig-verifier](https://github.com/dfinity/ic/blob/4c75016a882cd113dd41e924382b469deff7093c/rs/crypto/standalone-sig-verifier/Cargo.toml#L2) is not immediately clear to me, since this was also intended as a standalone sig verifier. Would it make sense to merge both?

Reply:
This new package (unlike the existing one) does not depend  on ic-repo-types, and can be published on crates.io

Reply:
I see, thanks for the explanation!